docs/root/configuration/cluster_manager/cluster_stats.rst

```diff
@@ -61,6 +61,7 @@ Every cluster has a statistics tree rooted at *cluster.<name>.* with the followi
   upstream_cx_none_healthy, Counter, Total times connection not established due to no healthy hosts
   upstream_rq_total, Counter, Total requests
   upstream_rq_active, Gauge, Total active requests
+  upstream_rq_error, Counter, Total requests that were reset after downstream response has already started.
```

Comment:
s/has already/had

Reply:
I know the host stat is named this, and honestly I can't remember why it's named that way, but can we consider a more descriptive name for this? Also, this description is very specific to HTTP which makes me think it's not very upstream generic. Should this instead be a router stat? Thoughts?

Reply:
I was about to respond to the other comment - I agree , I think calling this  `rq_error`  is not correct. Also for host it is incremented at multiple places like https://github.com/envoyproxy/envoy/blob/master/source/common/router/router.cc#L563 so it accounts for all errors and limiting this to just for this specific case and calling it `rq_error` seems incorrect.
Do we have router stats or are you suggesting to add a new stat hierarchy?

Reply:
We already have router stats, take a look at the router code files. It should be easy to add.

Reply:
Yeah. Got it. Will make changes to add it to router stats.